python/_brotli.cc

```diff
@@ -88,6 +87,50 @@ static int lgblock_convertor(PyObject *o, int *lgblock) {
   return 1;
 }
 
+static BROTLI_BOOL compress_stream(BrotliEncoderState* enc, BrotliEncoderOperation op,
+                                   std::vector<uint8_t>* output, uint8_t* input, size_t input_length) {
+  const size_t kBufferSize = 65536;
+  const size_t kBufferPadding = 10240;
+
+  size_t buffer_length;
+  if (input_length > 0) {
+    buffer_length = input_length + (input_length >> 2) + kBufferPadding;
+  } else {
+    input_length = 0;
+    input = NULL;
+    buffer_length = kBufferSize;
+  }
+  uint8_t* buffer = new uint8_t[buffer_length];
```

Comment:
I think using `BrotliEncoderTakeOutput` is preferable in wrappers workflow.
It has 2 useful features:
* choosing size, allocating and freeing intermediate output buffer is not required anymore
* one less memcpy (from internal brotli buffer to intermediate output buffer)